jetcd-core/src/main/java/io/etcd/jetcd/impl/ClientConnectionManager.java

```diff
@@ -73,7 +74,15 @@
         this.credential = new AuthCredential(this);
 
         if (builder.executorService() == null) {
-            this.executorService = Executors.newCachedThreadPool();
+            final ThreadFactory backingThreadFactory = Executors.defaultThreadFactory();
```

Comment:
can you put this into https://github.com/etcd-io/jetcd/blob/main/jetcd-core/src/main/java/io/etcd/jetcd/support/Util.java to reuse?

Reply:
Let me give it a try.

Reply:
Done. PTAL?